### PR TITLE
Update USB Button with recent changes.

### PR DESCRIPTION
**Pull request by robabram**
Added USBRequestCode Enum from USB documenation.  Added checks for USBRequestCode usage.

### CODE REVIEW

File: ultimarc/devices/usb_button.py [hunk 1]
```diff
@@ -50,16 +48,16 @@ def set_color(self, red, green, blue):
                 raise ValueError(_('Color argument value is invalid'))
 
         data = USBButtonColorStruct(0x01, red, green, blue)
-        return self.write(USBB_SET_COLOR, USBButtonReportID, USBButtonWIndex, data, ct.sizeof(data))
+        return self.write(USBRequestCode.SET_CONFIGURATION, 0x0, USBButtonWIndex, data, ct.sizeof(data))
```
Comment (katie-snow) [inline ultimarc/devices/usb_button.py:51]: Change back to using USBButtonReportID here for the 0x0.

File: ultimarc/devices/usb_button.py [hunk 2]
```diff
@@ -50,16 +48,16 @@ def set_color(self, red, green, blue):
...
     def get_color(self):
         """
         Return the current USB button RGB color values.
         :return: (Integer, Integer, Integer) or None
         """
-        data = USBButtonColorStruct()
+        data = USBButtonColorStruct(0x01, 0x0, 0x0, 0x0)
         for x in range(20):
-            ret = self.read(USBB_GET_COLOR, USBButtonReportID, USBButtonWIndex, data, ct.sizeof(data))
+            ret = self.read(USBRequestCode.CLEAR_FEATURE, 0x0, USBButtonWIndex, data, ct.sizeof(data))
```
Comment (katie-snow) [inline ultimarc/devices/usb_button.py:60]: Change back to USBButtonReportID for the 0x0

**Review by katie-snow — COMMENTED** [review]
Reviewed PR

File: ultimarc/devices/usb_button.py (hunk 1 above)
Comment (robabram) [reply ultimarc/devices/usb_button.py]: Done

File: ultimarc/devices/usb_button.py (hunk 2 above)
Comment (robabram) [reply ultimarc/devices/usb_button.py]: Done

**Review by katie-snow — APPROVED** [review]

**Pull request merged**